### PR TITLE
Reformatting of the single column blog list view 

### LINKED ISSUE #2487: Reformatting of the single column blog list view on layer5.io/blog.
**Description**
Currently, the blog views are in Grid view. we have to change it into a single column blog list view.

**Current Behaviour **
Screenshot (590)

 

**Expected Behavior**
 We have to change it into a single view list. 

**Screenshots**
 see the difference between the blog views? Single and grid?
Screen Shot 2022-01-16 at 10 13 10 AM

**Environment:**
- Host OS: 
- Browser: 

---

**Area*
 [layer5.io/blog](https://layer5.io/blog).

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by AbhishekJamhoriya**
Signed-off-by: AbhishekJamhoriya <jamhoriya.1@iitj.ac.in>

**Description**
changed the UI of the single column blog list view from broad to dense.
This PR fixes #2487

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Can you list the issue that this PR resolves?
The mentioned issue number is incorrect.

**Comment by l5io** [comment]
🚀 Preview for commit 6137da9413aa43c7b0a309692c5ca4269a2fa47e at: https://61f00919a3887219990682cd--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Ooooh, nice. This is looking pretty good, @AbhishekJamhoriya

**Comment by AbhishekJamhoriya** [comment]
Thanks @leecalcote

**Comment by AbhishekJamhoriya** [comment]
> Can you list the issue that this PR resolves? The mentioned issue number is incorrect.

Done.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
The cards are looking a bit weird in smaller views, could you please reduce the extra whitespace on the top and the bottom. 
Screenshot 2022-01-25 at 11 52 01 AM

File: src/sections/Blog/Blog-list/blogList.style.js [hunk 1]
```diff
@@ -5,48 +5,71 @@ export const BlogPageWrapper = styled.div`
...
 		}
 	}
 
 	.post-content-block{
 		height: 8.5rem;
+		width: 75%;
 	}
 
 	.post-title{
 		font-size: 1.75rem;
```
Comment (Nikhil-Ladha) [inline src/sections/Blog/Blog-list/blogList.style.js:40]: Let's make this 1.5rem

File: src/sections/Blog/Blog-list/index.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ const BlogList = ({
   setGridView,
   pageContext,
 }) => {
+  console.log("data",data);
```
Comment (Nikhil-Ladha) [inline src/sections/Blog/Blog-list/index.js:20]: Remove this

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Also, while we are on this page I think it is better to update the limit from 8 to 10 for blogs per page as we have a lot of tags now, which goes far below the blogs section. So, this will accommodate that.
Update this https://github.com/layer5io/layer5/blob/8d2b25cf4f3f5e68ce664e76868c81d97854c1b9/gatsby-node.js#L271
to 10.

**Comment by l5io** [comment]
🚀 Preview for commit bcae184b13ffaf944aef8bbe05760d9ba0526909 at: https://61f139a7e43c3ea68e15903e--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3d97517e0a89f55f2572254a54c4bf63ada47252 at: https://61f387881b85074641a257ee--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
LGTM 👍🏼

**Comment by Nikhil-Ladha** [comment]
@warunicorn19 do you wanna review or not 😂 .....otherwise please clear the path for merging :p

**Comment by warunicorn19** [comment]
So sorry I missed this @Nikhil-Ladha😅

**Pull request merged**